ai-platform-unified/notebooks/official/automl/automl-text-classification.ipynb

```diff
@@ -0,0 +1,767 @@
+{
```

Comment:
<blockquote>BUCKET_OUTPUT_FILE = "gs://automl-text-notebook-20210604194135/output/prediction-e2e-text-classification-model-2021-06-04T20:13:37.743910Z/"</blockquote><p>Suggest renaming the value to something like "[bucket-output-folder-uri]"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/360/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

Reply:
Done

Reply:
<p>Acknowledged.</p>